Assets/PatchKit/UnityEditorExtension/Editor/UI/CreateAppScreen.cs

```diff
@@ -109,6 +130,9 @@ public override void Draw()
 
     [SerializeField]
     private string _name;
+    
+    [SerializeField]
+    private Texture2D _arrowIcon;

```

Comment:
Please load the icon through `EditorGUIUtility.Load`, just like `AboutScreen` is loading logo.

Reply:
no problem, IMO, we should load this img in higher class (screen?) because now we're loading this everywhere